packages/components/src/internal/components/domainproperties/dataclasses/DataClassDesigner.tsx

```diff
@@ -464,23 +489,35 @@ export class DataClassDesignerImpl extends PureComponent<Props & InjectedBaseDom
                     domain={model.domain}
                     headerTitle="Fields"
                     helpTopic={null} // null so that we don't show the "learn more about this tool" link for this domains
-                    controlledCollapse={true}
-                    initCollapsed={currentPanelIndex !== 1}
-                    validate={validatePanel === 1}
+                    controlledCollapse
+                    initCollapsed={currentPanelIndex !== DOMAIN_PANEL_INDEX}
+                    validate={validatePanel === DOMAIN_PANEL_INDEX}
                     panelStatus={
-                        model.isNew ? getDomainPanelStatus(1, currentPanelIndex, visitedPanels, firstState) : 'COMPLETE'
+                        model.isNew
+                            ? getDomainPanelStatus(DOMAIN_PANEL_INDEX, currentPanelIndex, visitedPanels, firstState)
+                            : 'COMPLETE'
                     }
                     onChange={this.onDomainChange}
-                    onToggle={(collapsed, callback) => {
-                        onTogglePanel(1, collapsed, callback);
-                    }}
+                    onToggle={this.formToggle}
                     appPropertiesOnly={appPropertiesOnly}
                     useTheme={useTheme}
                     successBsStyle={successBsStyle}
                     testMode={testMode}
                     domainFormDisplayOptions={domainFormDisplayOptions}
                     systemFields={model.options.systemFields}
                 />
+                {appPropertiesOnly && !model.isBuiltIn && (
+                    <DataTypeProjectsPanel
+                        controlledCollapse
+                        dataType="DataClass"
```

Comment:
Yes, it can! I forgot it was on that object. I'll remove the `dataType` prop from `DataTypeProjectsPanel` and use the param from the entityDataType instead.